src/main/resources/lang/french.lang

```diff
@@ -0,0 +1,190 @@
+		main config: la configuration principale
+		aliases: les alias
+		script: <gold>%s<reset>
+		scripts in folder: tous les scripts dans <gold>%s<reset>
+		x scripts in folder: <gold>%2$s <reset>script¦¦s¦ dans <gold>%1$s<reset>
+		empty folder: <gold>%s<reset> ne contient aucun script activé.
+	enable:
+		all:
+			enabling: Activation de tous les scripts désactivés...
+			enabled: Activation et analysation réussis de tous les scripts précédemment désactivés.
+			error: %s ¦erreur rencontrée¦erreurs rencontrées¦ lors de l'analysation des scripts désactivés !
+			io error: Impossible de charger les scripts (certains scripts ont peut-être déjà été renommés et seront activés au redémarrage du serveur) : %s
+		single:
+			already enabled: <gold>%s<reset> est déjà activé ! Utilisez <gray>/<gold>skript <cyan>reload <red>%s<reset> pour le recharger s'il a été modifié.
+			enabling: Activation de <gold>%s<reset>...
+			enabled: <gold>%s<reset> activé et analysé avec succès.
+			error: %2$s ¦erreur rencontrée¦erreurs rencontrées¦ lors de l'analysation de <gold>%1$s<red> !
+			io error: Impossible d'activer <gold>%s<red>: %s
+		folder:
+			empty: <gold>%s<reset> ne contient aucun script désactivé.
+			enabling: Activation de <gold>%2$s <reset>script¦¦s¦ dans <gold>%1$s<reset>...
+			enabled: Activation et analysation réussis de <gold>%2$s<reset> script(s) précédemment désactivé(s) dans <gold>%1$s<reset>.
+			error: %2$s ¦erreur rencontrée¦erreurs rencontrées¦ lors de l'analysation des scripts dans <gold>%1$s<red> !
+			io error: Erreur lors de l'activation des scripts dans <gold>%s<red> (certains scripts pourraient être activés lors du redémarrage du serveur) : %s
+	disable:
+		all:
+			disabled: Désactivation de tous les scripts réussie.
+			io error: Impossible de renommer tous les scripts - certains scripts seront réactivés lorsque vous redémarrerez le serveur : %s
+		single:
+			already disabled: <gold>%s<reset> est déjà désactivé !
+			disabled: <gold>%s<reset> désactivé avec succès.
+			io error: Impossible de renommer <gold>%s<red>, il sera à nouveau activé lorsque vous redémarrerez le serveur : %s
+		folder:
+			empty: <gold>%s<reset> ne contient aucun script activé.
+			disabled: Désactivation réussie de <gold>%2$s<reset> script(s) dans <gold>%1$s<reset>.
+			io error: Impossible de désactiver les scripts <gold>%s<red> (certains scripts pourraient être désactivés lors du redémarrage du serveur): %s
+	update:
+		# check/download: see Updater
+		changes:
+			# multiple versions:
+			# 	title: <gold>%s<r> update¦ has¦s have¦ been released since this version (<gold>%s<r>) of Skript:
+			# 	footer: To show the changelog of a version type <gold>/skript update changes <version><reset>
+			# invalid version: No changelog for the version <gold>%s<red> available
+			title: <bold><cyan>%s<reset> (%s)
+			next page: <grey>page %s sur %s. Utilisez <gold>/skript update changes %s<gray> pour la page suivante (astuce : utilisez la touche flèche haut)
```

Comment:
Wouldn't it be better to say `flèche du haut` ?

Reply:
Yeah, you're right, sounds better.

Reply:
The modification has not been made

Reply:
Applied, thanks for pointing that out 😊